### PR TITLE
Fix appearance of panels showing caption

### PR DESCRIPTION
**Pull request by WatanabeToshimitsu**
## What
Fix appearance of panels showing caption

## Why 
使いやすいデザインにするため

## See also
https://github.com/dataware-tools/app-scene-viewer/pull/40

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/webviz/pull/7#" title="Author self-approved">WatanabeToshimitsu</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fwebviz).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dataware-tools/webviz/blob/dataware-tools/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by yusukefs — APPROVED** [review]
LGTM!

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/webviz/pull/7#" title="Author self-approved">WatanabeToshimitsu</a>*, *<a href="https://github.com/dataware-tools/webviz/pull/7#pullrequestreview-838024193" title="Approved">yusukefs</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fwebviz).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/dataware-tools/webviz/blob/dataware-tools/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
